xml/System.Drawing.Drawing2D/HatchStyle.xml

```diff
@@ -831,7 +831,13 @@
       </ReturnValue>
       <MemberValue>4</MemberValue>
       <Docs>
-        <summary>Specifies hatch style <see cref="F:System.Drawing.Drawing2D.HatchStyle.SolidDiamond" />.</summary>
+        <summary>Specifies hatch style <see cref="F:System.Drawing.Drawing2D.HatchStyle.LargeGrid" />.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+## Remarks  
+ This value differs from the definition of HatchStyleMax in the GDI+ native headers, which is defined as HatchStyleTotal - 1.
+ ]]></format>
+        </remarks>
```

Comment:
Is this remark ever shown somewhere? I don't see it on the [preview ](https://review.docs.microsoft.com/en-us/dotnet/api/system.drawing.drawing2d.hatchstyle?view=dotnet-plat-ext-2.1&branch=pr-en-us-6622)of this doc page.

cc @gewarren.

Reply:
There is a limitation currently in that enum remarks aren't shown anywhere (hence the build warning). However, we have a [feature request](https://ceapex.visualstudio.com/Engineering/_workitems/edit/390259) for enum fields to have their own pages in docs, in which case remarks and applicable versions will be shown. For now, consider putting this info into the summary. cc @mimisasouvanh .